### ISSUE TITLE
Incorrect license headers

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/alba-asdmanager#111]
http://testrail.openvstorage.com/index.php?/tests/view/1087146&group_by=cases:section_id&group_order=asc&group_id=29863

```
Following files were found with different licenses:
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/__init__.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/tx.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/nursery.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/compat.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/utils.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/consistency.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/test.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/sequence.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/errors.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/protocol/__init__.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/protocol/admin.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/client/__init__.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/client/admin.py
- /opt/asd-manager/source/tools/pyrakoon/pyrakoon/client/utils.py
- /opt/asd-manager/source/tools/configuration/__init__.py
```

**Comment by khenderick** [comment]
Only the configuration's ```__init__.py``` file is missing. The rest is pyrakoon, an external library which license can be ignored.

**Comment by khenderick** [comment]
Fixed by #112, packaged in openvstorage-sdm-1.6.3-rev.357.ee7c9f5

**Comment by JeffreyDevloo** [comment]
### Steps
- Installed latest ovs
- Ran the autotest that gave the result

### Output

```
In [4]: TestAfterCare.check_license_headers_test()
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-4-30111295b25c> in <module>()
----> 1 TestAfterCare.check_license_headers_test()

/opt/OpenvStorage/ci/tests/system/validation_after_test.pyc in check_license_headers_test()
    353 
    354         for storagerouter in storagerouters:
--> 355             assert len(files_with_diff_licenses[storagerouter.guid]) == 0, 'Following files were found with different licenses:\n - {0}'.format('\n - '.join(files_with_diff_licenses[storagerouter.guid]))

AssertionError: Following files were found with different licenses:
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/sequence.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/tx.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/__init__.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/errors.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/compat.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/consistency.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/nursery.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/utils.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/test.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/client/admin.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/client/__init__.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/client/utils.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/protocol/admin.py
 - /opt/asd-manager/source/tools/pyrakoon/pyrakoon/protocol/__init__.py
```
Pyrakoon is an external library of which we don't control the headers.

### Test result
Test passed.

### Setup

Hyperconverged setup

- Three nodes with each three disks for the back-end

### Package information

- openvstorage                       2.7.3-rev.3990.68491c4-1                                 amd64        openvStorage
- openvstorage-backend               1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.3-rev.3990.68491c4-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-health-check          2.0.0-rev.134.9ef41c0-1                                  amd64        Open vStorage HealthCheck
- openvstorage-sdm                   1.6.3-rev.357.ee7c9f5-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-test                  2.7.3-rev.1012.608f550-1                                 amd64        openvStorage autotest suite
- openvstorage-webapps               2.7.3-rev.3990.68491c4-1                                 amd64        openvStorage Web Applications
